.github/workflows/pr.yml

```
@@ -38,7 +38,7 @@ jobs:
           output_path: 'results'
           platform_type: terraform
           fail_on: high,medium
-          exclude_queries: 8e94dced-9bcc-4203-8eb7-7e41202b2505 # Excludes 'Auto Scaling Group With No Associated ELB' query
+          exclude_queries: '8e94dced-9bcc-4203-8eb7-7e41202b2505,e592a0c5-5bdb-414c-9066-5dba7cdea370' # Excludes 'Auto Scaling Group With No Associated ELB' query
```

Small thing but might be worth adding a comment for the second excluded query too.
```suggestion
          exclude_queries: '8e94dced-9bcc-4203-8eb7-7e41202b2505,e592a0c5-5bdb-414c-9066-5dba7cdea370' # Excludes 'Auto Scaling Group With No Associated ELB' and 'IAM Access Analyzer Undefined' queries
```

---

Agreed, updated